dev/content/pages/About.md

```diff
@@ -1,29 +1,21 @@
-Title: Hi I'm Ravin
+Title: Hi, I'm Ravin
 Date: 2016-4-21 
 Slug:  index
 Authors: Ravin Kumar
 Status: Published
 save_as: index.html 
 
-I strive to be
 
-1. Really good at math, particularly computational math
+I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.
+
+There are 3 types of people I strive to be:
```

Comment:
Three types of people, or three things?

Reply:
I wasn't sure if "There are 3 things I strive to be" sounded weird or not so I ended up trying to be more formal/grammatically correct. Idk if I even accomplished that, so I changed it to "things" lol. Looks better now.
![image](https://user-images.githubusercontent.com/20257416/135034883-77111582-07ff-4cea-a092-080c22f4b2dd.png)